test/integration/targets/listen_ports_facts/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+# Test playbook for the listen_ports_facts module
+# (c) 2019, Nathan Davison <ndavison85@gmail.com>
+
+# This file is part of Ansible
```

Comment:
Please use the one-line license statement (otherwise our code would be 30% license statements?
```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```